acceptance/indy_acceptance/indy-tests/test_results/Acceptance_Test_20171117_17-42-58/summary.html

```diff
@@ -0,0 +1,293 @@
+<html>
```

Comment:
It looks like a log. I think it needs to be removed

Reply:
We removed it.